portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/portalconfiguration/systemmanagement/OnDemandAdmin.testcase

```diff
@@ -0,0 +1,63 @@
+@component-name = "portal-user-management-and-portal-configuration"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Virtual Instances";
```

Comment:
Move the test to [On Demand Admin](https://testray.liferay.com/zh/home/-/testray/case_results?testrayComponentId=1616925690&testrayTeamId=1310700616&testrayBuildId=1878363714) on testray.